assets/src/domain/eventEditor/ui/ticketAssignmentsManager/filters/FilterBar.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import {
+	DatesByMonthControl,
+	ShowExpiredTicketsControl,
+	ShowTrashedDatesControl,
+	ShowTrashedTicketsControl,
+} from './controls';
+import { useTAMContext } from '../context';
+
+const FilterBar = () => {
+	const {
+		assignmentType,
+		filterState: {
+			datesByMonth,
+			setDatesByMonth,
+			setShowExpiredTickets,
+			setShowTrashedDates,
+			setShowTrashedTickets,
+			showExpiredTickets,
+			showTrashedDates,
+			showTrashedTickets,
+		},
+	} = useTAMContext();
+
+	return (
+		<div className='filter-bar'>
```

Comment:
it's not just a `filter-bar`, it's `ee-filter-bar` :)

Reply:
Done